oracle/src/sponge.rs

```diff
@@ -199,6 +203,10 @@ where
         P::ScalarField::from_repr(x.into()).unwrap_or_else(P::ScalarField::zero)
     }
 
+    fn digest_fq(mut self) -> P::BaseField {
```

Comment:
just `digest` since it's implemented for SpongeFq

Reply:
There's already a `digest`, called it this to avoid clobbering it. See the next review comment.